DNN Platform/DotNetNuke.Web/InternalServices/ControlBarController.cs

```diff
@@ -637,6 +651,13 @@ private int DoAddExistingModule(int moduleId, int tabId, string paneName, int po
                 }
                 else
                 {
+                    //copy tab module settings
```

Comment:
TabSettings need to be copied in both cases (but with different ModuleID, as they are reference specific.